egs/tedlium3/ASR/pruned_transducer_stateless/beam_search.py

```diff
@@ -88,7 +88,7 @@ def fast_beam_search(
         # (shape.NumElements(), 1, encoder_out_dim)
         # fmt: off
         current_encoder_out = torch.index_select(
-            encoder_out[:, t:t + 1, :], 0, shape.row_ids(1)
+            encoder_out[:, t:t + 1, :], 0, shape.row_ids(1).long()
```

Comment:
Would you please also fix this in other recipes, and it will be good to add some documents here (i.e. why we need `long()`).